iam_group.tf

```diff
@@ -32,10 +32,34 @@ data "aws_iam_policy_document" "bod_lambda_log_doc" {
   }
 }
 
-# The CloudWatch log policy for our IAM group
+# IAM policy document that allows listing of CloudWatch log groups.
+# This will be applied to the IAM group we are creating.
+data "aws_iam_policy_document" "list_log_doc" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:DescribeLogGroups",
+    ]
+
+    resources = [
+      "*",
```

Comment:
Is the comma supposed to be here is there is only one item in the list?

Reply:
It can be there or not.  When I notice it I like to add one, since then if I add another list item I can't forget to tack on a comma to what is not the penultimate item.

Reply:
👍